etc/login.defs.test

```diff
@@ -0,0 +1,473 @@
+#
+# /etc/login.defs - Configuration control definitions for the shadow package.
+#
+#	$Id$
+#
```

Comment:
Maybe we should have a tests/etc/ subdirectory?  I'm not very happy mixing actual stuff and tests.

Reply:
Yes, I'd also prefer to ship it under the tests folder. What about `tests/system/config/login.defs`? It's highly unlikely that this file will have any use in bash system tests and unit tests. In addition, we can use the folder to store any configuration file needed for python system tests.

Reply:
> Yes, I'd also prefer to ship it under the tests folder. What about `tests/system/config/login.defs`?

The usual spelling of .../system/config/ is `.../etc/`.  I prefer etc for consistency with the FHS.
<https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html>

In fact, GNU coding standards say that build systems should use the `$sysconfdir` variable, and define it to `$(prefix)/etc`.
<https://www.gnu.org/prep/standards/html_node/Directory-Variables.html#Directory-Variables>

Reply:
In this case system refers to system tests not system config. I'm fine with calling it `tests/system/etc/login.defs`